components/Slider/src/MDCSlider.h

```diff
@@ -292,6 +292,20 @@ IB_DESIGNABLE
  */
 @property(nonatomic, assign, getter=isThumbHollowAtStart) BOOL thumbHollowAtStart;
 
+/**
+Tells the accessibility element what amount to increment the value of its content.
+
+ Default increment level is 10%.
+ */
+@property(nonatomic) CGFloat accessibilityIncrementAmount;
```

Comment:
I don't see anything like this on UISlider, and I believe users can activate the slider to get smaller adjustments back toward the initial value. Is there a reason we think this API is needed specifically in MDCSlider?